src/cli.rs

```diff
@@ -293,6 +295,33 @@ pub enum PlotKind {
         #[structopt(long = "sample", required = true, help = "Sample to plot.")]
         sample: String,
     },
+    #[structopt(
+        name = "scatter",
+        about = "Plot variant allelic fraction scatter plot overlayed with a contour plot between two sample groups",
+        usage = "varlociraptor plot scatter --somatic-tumor-events SOMATIC_TUMOR \
+        --tumor-sample tumor --normal-sample normal < calls.bcf | vg2svg > scatter.svg",
+        setting = structopt::clap::AppSettings::ColoredHelp,
+    )]
+    Scatter {
+        #[structopt(
+            long = "somatic-tumor-events",
+            default_value = "SOMATIC_TUMOR",
```

Comment:
Here as well, keep it generic, not implying that this shall be only used for somatic variants. Although it of course makes most sense there, but nevertheless.